vendor/github.com/turbonomic/turbo-go-sdk/pkg/service/tap_service.go

```diff
@@ -70,6 +70,12 @@ func (tapService *TAPService) addTarget(isRegistered chan bool) {
 
 func (tapService *TAPService) ConnectToTurbo() {
 	glog.V(4).Infof("[ConnectToTurbo] Enter ******* ")
+
+	if tapService.RESTClient == nil {
+		glog.V(1).Infof("TAP service cannot be started - cannot create target")
```

Comment:
It should be an error message, isn't it?

Reply:
This is a change in the turbo-go-sdk and should be handled there.